src/modules/billing/routes/batches.js

```diff
@@ -125,13 +125,15 @@ const getBatchInvoiceDetail = {
 
 const deleteBatchInvoice = {
   method: 'DELETE',
-  path: `${BASE_PATH}/{batchId}/invoices/{invoiceId}`,
+  path: `${BASE_PATH}/{batchId}/invoices/{invoiceId}/org/{originalInvoiceId}/rebill/{rebillInvoiceId}`,
   handler: controller.deleteBatchInvoice,
   config: {
     validate: {
       params: Joi.object().keys({
         batchId: Joi.string().uuid().required(),
-        invoiceId: Joi.string().uuid().required()
+        invoiceId: Joi.string().uuid().required(),
+        originalInvoiceId: Joi.string().uuid(),
+        rebillInvoiceId: Joi.string().uuid()
```

Comment:
originalInvoiceId and rebillInvoiceId should be added as Joi.string().uuid().optional() query parameters.